src/server/api/routers/public.ts

```diff
@@ -0,0 +1,31 @@
+import {
+    createTRPCRouter,
+    publicProcedure,
+  } from "@/server/api/trpc";
+  import { Exhibitor } from "@/shared/Classes";
+  
+  export const publicRouter = createTRPCRouter({
+    getExhibitors: publicProcedure.query(async ({ ctx }) => {
+  
+        const exhibitors = await ctx.prisma.exhibitor.findMany();
+        return exhibitors.map(
+          (exhibitor) =>
+            new Exhibitor(
+              exhibitor.id,
+              exhibitor.name,
+              exhibitor.organizationNumber,
+              exhibitor.invoiceEmail,
+              exhibitor.logoWhite?.toString("base64"),
+              exhibitor.logoColor?.toString("base64"),
+              exhibitor.description,
+              exhibitor.package,
+              exhibitor.extraTables,
+              exhibitor.extraChairs,
+              exhibitor.extraDrinkCoupons,
+              exhibitor.extraRepresentativeSpots,
+              exhibitor.totalBanquetTicketsWanted,
+              exhibitor.jobOfferId
```

Comment:
It's probably not critical, but we should not send information that we don't use to the client here. For instance the extra orders and invoice email should perhaps not be shared publicly.

Reply:
Yes, definitely. Switched to only transmit id, name, logos, desciption, package, and jobofferid intead

Reply:
Nice! This file is unused now, right?

Reply:
Yep, ur right. Fixed now.